src/ownership/lifetimes-function-calls.md

```diff
@@ -23,3 +23,12 @@ fn main() {
 * Read `&'a Point` as "a borrowed `Point` which is valid for at least the
   lifetime `a`".
   * The _at least_ part is important when parameters are in different scopes.
+
+<details>
+
+In the above example, try the following:
+
+* Move the declaration of `p2` and `p3` into a a new scope (`{ ... }`). You have to declare `p3` as `let p3: &Point;`, in `main`'s scope before that new block, and in the block assign to `p3` without the `let` (just `p3 = left_most(...)`). Note how this does not compile since `p3` outlives `p2`.
```

Comment:
Just a note: you could write full code blocks here — it's normal Markdown syntax. I think it would make the explanation both shorter and clearer.

Reply:
Makes a lot of sense, thanks for the suggestion.